huaweicloud/services/cbh/resource_huaweicloud_cbh_instance.go

```diff
@@ -504,6 +631,14 @@ func resourceCBHInstanceUpdate(ctx context.Context, d *schema.ResourceData, meta
 		return diag.Errorf("error creating CBH client: %s", err)
 	}
 
+	// If `power_action` is changed from stop to start, the instance should be startup first.
```

Comment:
please describe the reason why the `Start UP` operation must be performed before other parameters are updated. This is the focus of the reviewer.

Reply:
done.